db/migrate/20220624090353_add_state_to_pull_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToPullRequests < ActiveRecord::Migration[7.0]
+  def change
+    add_column :github_pull_requests, :state, :tinyint, default: 1, null: false, if_not_exists: true
```

Comment:
I've set the default to `1` as that is sort of what all current prs are. We'll do a post merge script to update the state of all pull requests.